### PR TITLE
KOGITO-618 Changed logic for filtering data

### PR DESCRIPTION
**Pull request by AjayJagan**
- on Click of Apply Filter button the data is re-fetched every time.
- Initially the parentProcessInstanceId whose state is 'ACTIVE' is loaded, instead of loading the entire data

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@AjayJagan I noticed that now when I select a filter in the dropdown, it automatically updates the status panel on the right side. I guess that should only be update once I apply the filter? @srambach wdyt?

File: packages/management-console/server/server.js [hunk 1]
```diff
@@ -730,7 +730,12 @@ const resolvers = {
                 console.log("data", datum.parentProcessInstanceId);
                 if (args["filter"].id) {
                   return (datum.id == args["filter"].id);
-                } else {
+                }
+                else if (args["filter"].parentProcessInstanceId[0] == null){
```
Comment (cristianonicolai) [inline packages/management-console/server/server.js]: please align else statements
```suggestion
               } else if (args["filter"].parentProcessInstanceId[0] == null){
```

**Comment by AjayJagan** [comment]
> @AjayJagan I noticed that now when I select a filter in the dropdown, it automatically updates the status panel on the right side. I guess that should only be update once I apply the filter? @srambach wdyt?

sure that can be done.

**Comment by srambach** [comment]
Yes, I agree - the chip group should always indicate what filter is currently being applied to the list. So it wouldn't update until the button is pushed to apply the filter.
@cristianonicolai @AjayJagan

File: packages/management-console/server/server.js (hunk 1 above)
Comment (AjayJagan) [reply packages/management-console/server/server.js]: Should an else statement be addded?

Comment (cristianonicolai) [reply packages/management-console/server/server.js]: its just the formatting that looks a bit odd.

**Comment by AjayJagan** [comment]
> Yes, I agree - the chip group should always indicate what filter is currently being applied to the list. So it wouldn't update until the button is pushed to apply the filter.
> @cristianonicolai @AjayJagan

Yes. Now the chips would be shown only on click of the Apply Filter button.

File: packages/management-console/server/server.js (hunk 1 above)
Comment (AjayJagan) [reply packages/management-console/server/server.js]: I guess I will add the prettier soon :) . That should solve most of the formatting issue.

Comment (cristianonicolai) [reply packages/management-console/server/server.js]: you could just fix these ones quite easily, the prettier formatting can be done later

**Review by cristianonicolai — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Pull request merged**